### PR TITLE
Add script to fix H1s on Dokka-generated docs

### PR DESCRIPTION
**Pull request by cbush**
This is an SEO team request to make sure all pages have an H1.

For more info see https://jira.mongodb.org/browse/DOCSP-31274

This has not been fully tested due to my lack of environment that can successfully build the Dokka files. Please validate.

### CODE REVIEW

File: tools/fix-h1s.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -ex
+
+# Assume Dokka has been run
+pushd "`dirname "$0"`"/../docs/html
```
Comment (cbush) [inline tools/fix-h1s.sh:4]: Please confirm this is the output path of the dokka command.

File: tools/fix-h1s.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+# Make the output SEO friendly by converting the "h2" title to the proper "h1".
+# For Dokka, this is only an issue on the index page (apparently).
+sed -i -e 's|<h2\(.*\)</h2>|<h1\1</h1>|' index.html
```
Comment (cbush) [inline tools/fix-h1s.sh:8]: Goal is to change this h2 to an h1.
image

Comment (rorbech) [reply tools/fix-h1s.sh:19]: I have verified the updated script by running 
```
../tools/fix-h1s.sh build/dokka/htmlMultiModule
``` 
inside `packages`. 

It now yields the desired `h1` headings:
Screenshot 2023-07-25 at 10 41 48

**Review by rorbech — APPROVED** [review]
I have made some minor adjustments and verified it in my development environment.

**Comment by cbush** [comment]
Thank you!

**Pull request merged**